examples/compute_shader.rs

```diff
@@ -26,60 +26,59 @@ fn create_static_sampler(
 }
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    unsafe {
```

Comment:
View this with https://github.com/Traverse-Research/saxaboom/pull/20/files?diff=unified&w=1 - turns out the only `unsafe` thing was the `union` field access because we guard the rest of the C API.